.gitattributes

```diff
@@ -0,0 +1,35 @@
+# Code
```

Comment:
.gitattributes needed so that git can handle the different line endings between windows and linux.